jetty-io/src/main/java/org/eclipse/jetty/io/ManagedSelector.java

```diff
@@ -60,6 +61,20 @@
 public class ManagedSelector extends ContainerLifeCycle implements Dumpable
 {
     private static final Logger LOG = Log.getLogger(ManagedSelector.class);
+    private static final boolean FORCE_SELECT_NOW;
+    static
+    {
+        String property = System.getProperty("org.eclipse.jetty.io.forceSelectNow");
+        if (property != null)
+        {
+            FORCE_SELECT_NOW = Boolean.parseBoolean(property);
+        }
+        else
+        {
+            property = System.getProperty("os.name");
+            FORCE_SELECT_NOW = property != null && property.toLowerCase(Locale.ENGLISH).contains("windows");
```

Comment:
s/contains("windows")/startsWith("windows")

Reply:
I think `contains()` is more generic, and we've been bitten in the past by being too strict on checking system properties.
If really there is an ambiguity, it can be resolved by setting explicitly the system property.

Reply:
All the examples I could find online and in other parts of Jetty code use `contains()`.
I doubt there are other OS with the name Windows in it, it would be a TM violation.